general/getting-started-in-tech.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: page
+title: Getting Started in Tech
+---
+
+Technology is a vast subject. It can enable you to write a to-do list on your phone, sell goods online or send a space probe to Neptune. Software is built by developers and codebar is here to teach you how to write such software. Whether you want to make a career of it or use your skills on personal projects.
```

Comment:
```suggestion
Technology is a vast subject. It can enable you to write a to-do list on your phone, sell goods online or send a space probe to Neptune. Software is built by developers and codebar is here to teach you how to write such software, whether you want to make a career of it or acquire some exta skills.
```
I think we should avoid the user of the personal projects wording, given it's something we tend to have problems with at workshops!

Reply:
very true 👍